packages/server/src/__tests__/logger.test.ts

```diff
@@ -1,4 +1,4 @@
-import { ApolloServer } from '../..';
```

Comment:
This is weird, right? Was this just wrong before?

Reply:
I think it probably worked before because of, I dunno, something in the ../.. package.json that made it find the right thing?